Jenkinsfile

```diff
@@ -20,6 +20,18 @@ pipeline {
   }
 
   stages {
+    stage('Deps') {
+      steps {
+        sh "yarn install --frozen-lockfile"
+      }
+    }
+
+    stage('Checks') {
```

Comment:
@jakubgs how do I add this step as part of a pull request check please?

Reply:
One way would be through GitHub Actions. We could do it in Jenkins, but that would require a separate `Jenkinsfile` that runs just the checks and updates github PR with check results.

Reply:
I wasn't too sure if we want to mix GitHub actions and Jenkins for CI jobs but it makes sense to use GH actions as it is a quick/low resource job. Thanks!

Reply:
I'll setup the GH action as part of this PR.

Reply:
@jakubgs can you please enable github action on this repo or provide me with admin rights so I can do it? Cheers.